src/style.css

```diff
@@ -0,0 +1,37 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  background-color: #000;
+  margin: 20px;
+}
+
+.big-box {
+  width: 350px;
+  margin-bottom: 40px;
+  padding: 10px;
+  background-color: #308dc0;
+  border: 1px solid #e95d49;
+}
+
+.middle-box {
+  padding: 10px;
+  background-color: #ffa500;
+  border: 1px solid #e95d49;
+}
+
+.little-box {
+  padding: 10px;
+  background-color: #ddd;
+  border: 1px solid #e95d49;
+}
+
+div {
+  border: 1px solid #e95d49;
```

Comment:
You can also make 1 more class for all the boxes, and write border: 1px solid #e95d49 for each box. Make your code DRY. Then each box would have 2 classes (e.g.  div class="box little-box")